packages/renderer/src/lib/container/ContainerActions.svelte

```diff
@@ -43,49 +45,60 @@ function deployToKubernetes(): void {
 }
 </script>
 
-<ListItemButtonIcon
-  title="Open Browser"
-  onClick="{() => openBrowser(container)}"
-  hidden="{!(container.state === 'RUNNING' && container.hasPublicPort)}"
-  backgroundColor="{backgroundColor}"
-  icon="{faExternalLinkSquareAlt}" />
-<ListItemButtonIcon
-  title="Open Terminal"
-  onClick="{() => openTerminalContainer(container)}"
-  hidden="{!(container.state === 'RUNNING')}"
-  backgroundColor="{backgroundColor}"
-  icon="{faTerminal}" />
-<ListItemButtonIcon
-  title="Generate Kube"
-  onClick="{() => openGenerateKube()}"
-  hidden="{!(container.engineType === 'podman' && container.groupInfo.type === ContainerGroupInfoTypeUI.STANDALONE)}"
-  backgroundColor="{backgroundColor}"
-  icon="{faFileCode}" />
-<ListItemButtonIcon
-  title="Deploy to Kubernetes"
-  onClick="{() => deployToKubernetes()}"
-  hidden="{!(container.engineType === 'podman' && container.groupInfo.type === ContainerGroupInfoTypeUI.STANDALONE)}"
-  backgroundColor="{backgroundColor}"
-  icon="{faRocket}" />
 <ListItemButtonIcon
   title="Start Container"
   onClick="{() => startContainer(container)}"
   hidden="{container.state === 'RUNNING'}"
   backgroundColor="{backgroundColor}"
   icon="{faPlayCircle}" />
+
 <ListItemButtonIcon
   title="Stop Container"
   onClick="{() => stopContainer(container)}"
   hidden="{!(container.state === 'RUNNING')}"
   backgroundColor="{backgroundColor}"
   icon="{faStopCircle}" />
-<ListItemButtonIcon
-  title="Restart Container"
-  onClick="{() => restartContainer(container)}"
-  backgroundColor="{backgroundColor}"
-  icon="{faArrowsRotate}" />
-<ListItemButtonIcon
-  title="Delete Container"
-  onClick="{() => deleteContainer(container)}"
-  backgroundColor="{backgroundColor}"
-  icon="{faTrash}" />
+
+<!-- If dropdownMenu is true, use it, otherwise just show the regular buttons -->
+<DropdownMenu backgroundColor="{backgroundColor}" enabled="{dropdownMenu}">
```

Comment:
so thinking again, here I think we should use
```
{#if dropdownMenu}
  <DropDownMenu backgroundColor="{backgroundColor}"
{/if}
```
the enable/disable state should be a state where the drop down is displayed but disabled (like you can't click on it, it appears using a light grey for example)